package.json

```diff
@@ -75,7 +75,7 @@
     "@angular/bazel": "^10.1.0",
     "@angular/benchpress": "^0.2.1",
```

Comment:
Can we change `@angular/benchpress` to `0.2.1` rather than `^0.2.1` so that it matches `@angular/dev-infra-private`?